SofaKernel/modules/SofaCore/src/sofa/core/topology/Topology.h

```diff
@@ -37,21 +37,22 @@ namespace topology
 {
 
 /// The enumeration used to give unique identifiers to Topological objects.
-enum TopologyObjectType
+enum TopologyElementType
```

Comment:
[**enum class**](https://www.stroustrup.com/C++11FAQ.html#enum) should be preferred when it is possible

Reply:
yes but... if I do that change I get 11937 errors....